src/card.css.ts

```diff
@@ -341,3 +341,27 @@ export const videoAction = style([
         },
     },
 ]);
+
+const aspectRatio = createVar();
+
+export const vars = {aspectRatio};
+
+export const cardContainer = style({
+    position: 'relative',
+    aspectRatio,
+    '@supports': {
```

Comment:
Maybe I would check if the browser supports the aspect ratio using JS, like **CSS.supports()**. Because it has better coverage on Safari. "@supports" just works from Safari 14 and "CSS.supports" goes until Safari 9.

CSS.supports() (https://caniuse.com/?search=CSS.supports)
@supports (https://caniuse.com/?search=@supports)

Reply:
hmmm... no, you're wrong, @supports in css is supported in safari 9 https://caniuse.com/css-featurequeries